components/Quote/Quote.module.css

```diff
@@ -0,0 +1,28 @@
+.largecontainer,
+.container {
+	margin: 1em 0;
+	padding: 1em;
+    border-left-style: outset;
+    color: gray;
+}
+
+.largecontainer p {
+    font-size: 3rem;
+}
+
+.container cite,
+.container p {
+    display: inline;
+}
+
+.largecontainer cite {
+    display: block;
+    text-align: right;
+}
+
+.largecontainer cite:before,
+.container cite:before {
+    padding-right: 0.5em;
+    padding-left: 2em;
+    content: ' \2014 ';
```

Comment:
I think you can use the literal Unicode character. Does that not work?

Reply:
It does, just wasn't my first choice :-)